src/main/java/org/jvnet/hudson/test/TemporaryDirectoryAllocator.java

```diff
@@ -62,7 +70,7 @@ public TemporaryDirectoryAllocator() {
      */
     public synchronized File allocate() throws IOException {
         try {
-            File f = File.createTempFile("jenkins", "test", base);
+            File f = File.createTempFile("jenkins", (withoutSpace ? "test" : " test"), base);
```

Comment:
Just a nit that starting or ending with space is a bit annoying for listing and copy/paste.
I would prefer something like "a test"

Reply:
After #111, I would suggest `jkh` → `j h`.

Reply:
Thanks @jglick. I've merged master in my branch, and made the change you've suggested (the `"jkh"` prefix becoming `"j h"`).